controllers/rcj_soccer_referee_supervisor/referee/referee.py

```diff
@@ -25,13 +25,27 @@ def check_progress(self):
             self.progress_chck[robot].track(pos)
 
             if not self.progress_chck[robot].is_progress(robot):
-                print(f'Robot {robot}: Lack of progress')
+                self.log.event(
+                    supervisor=self,
+                    type="lack_of_progress",
```

Comment:
Create special file `choices.py` (or use consts.py if you think it's better). Add Event types there. I would also use uppercase letters for event types.
```python
from enum import Enum

class LogEventChoices(Enum):
     LACK_OF_PROGRESS = "LACK_OF_PROGRESS"
```
```suggestion
                    type=LogEventChoices.LACK_OF_PROGRESS.value,
```